Changes to use build bot configuration options. This includes a chang…

gregrodgers opened the pull request:
…e to use ninja as the default.  To revert to make, set AOMP_USE_NINJA=0

gregrodgers committed:
Changes to use build bot configuration options. This includes a change to use ninja as the default.  To revert to make, set AOMP_USE_NINJA=0

skatrak commented on trunk/build_project.sh:
```diff
@@ -32,19 +32,27 @@ fi
+#  These options taken from build bot cause build fails.
+#  So for now we are not using them.
+#-DCMAKE_C_COMPILER_LAUNCHER=ccache
+#-DCMAKE_CXX_COMPILER_LAUNCHER=ccache
```
These options may fail because the `ccache` package must be installed first (`apt install ccache`). It's quite good for speeding up the compilation process, but it's not necessary.

gregrodgers committed:
add support for ccache

gregrodgers committed:
update prerequisite instructions to include ccache

gregrodgers committed:
add the ability to build ninja as an AOMP supplemental component

gregrodgers commented:
Thank you Sergio,

I put a check in for ccache with the ability to turn it off with AOMP_USE_CCACHE.    I also added it to the list of prerequisites in the install documentation. 

I also added the ability to find ninja as an AOMP supplemental component.   So if you dont have ninja, simply run "./build_supp.sh ninja"
